parcel.go

```diff
@@ -13,48 +13,103 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	r, err := s.db.Exec(`
+	INSERT INTO parcel (client, status, address, created_at) 
+	VALUES (?, ?, ?, ?)
+	`, p.Client, p.Status, p.Address, p.CreatedAt)
+
+	if err != nil {
+		return 0, err
+	}
+
+	if id, err := r.LastInsertId(); err != nil {
+		return 0, err
+	} else {
+		return int(id), nil
+	}
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
+	r := s.db.QueryRow(`
+	SELECT * FROM parcel
+	WHERE number = ? 
+	LIMIT 1
+	`, number)
 
-	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
 
+	err := r.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
+
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
-	// реализуйте чтение строк из таблицы parcel по заданному client
-	// здесь из таблицы может вернуться несколько строк
-
-	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	rows, err := s.db.Query(`
+	SELECT * FROM parcel
+	WHERE client = ?
+	`, client)
+
+	if err != nil {
+		return res, err
```

Comment:
Если ошибка, то лучше возвращать `nil`
```suggestion
		return nil, err
```